spec/models/organization_spec.rb

```diff
@@ -394,16 +414,16 @@
 
   describe 'valid_items' do
     it 'returns an array of item partner keys' do
-      item = organization.items.first
+      item = create(:item, organization: organization)
       expected = { name: item.name, id: item.id, partner_key: item.partner_key }
       expect(organization.valid_items.count).to eq(organization.items.count)
       expect(organization.valid_items).to include(expected)
     end
-    it 'only shows active valid items' do
-      intial_count = organization.valid_items.count
-      organization.items.last.update(active: false)
-      final_count = organization.valid_items.count
-      expect(intial_count).to_not eq(final_count)
+
+    # NOTE: is this test needed? seems duplicative of the below
+    it 'only shows active items' do
+      expect { create(:item, :inactive, organization: organization) }.to change { organization.valid_items.count }.by(0)
+      expect { create(:item, :active, organization: organization) }.to change { organization.valid_items.count }.from(0).to(1)
```

Comment:
its this test needed? test right below seems to do basically the same thing.

```rb
    context 'with invisible items' do
      let!(:organization) { create(:organization, skip_items: true) }
      let!(:item1) { create(:item, organization: organization, active: true, visible_to_partners: true) }
      let!(:item2) { create(:item, organization: organization, active: true, visible_to_partners: false) }
      let!(:item3) { create(:item, organization: organization, active: false, visible_to_partners: true) }
      let!(:item4) { create(:item, organization: organization, active: false, visible_to_partners: false) }

      it 'only shows active and visible items' do
        expect(organization.valid_items).to eq([{ id: item1.id, partner_key: item1.partner_key, name: item1.name }])
      end
    end

```

Reply:
Kill it! :hocho: